.travis.yml

```diff
@@ -75,10 +75,10 @@ jobs:
       - CHROMATIC_APP_CODE="m1dh5kc7oj"
     deploy:
       - provider: script
-        script: npm config set //registry.npmjs.org/:_authToken=$NPM_PUBLISH_TOKEN && node utils/publish-canary.js $SLACK_WEBHOOK $TRAVIS_BUILD_WEB_URL
+        script: npm config set //registry.npmjs.org/:_authToken=$NPM_PUBLISH_TOKEN && node utils/publish-canary.js
         skip_cleanup: true
         on:
-          branch: master
+          all_branches: true
```

Comment:
Just realized - Is this going to make the scripts above run on every branch? What keeps those scripts locked down to master currently?

Reply:
https://github.com/Workday/canvas-kit/pull/481/files#diff-354f30a63fb0907d4ad57269548329e3R26-R27